src/GovUk.Education.ExploreEducationStatistics.Content.Model/EmbedBlock.cs

```diff
@@ -0,0 +1,24 @@
+﻿#nullable enable
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model;
+
+public class EmbedBlock
+{
+    public Guid Id { get; set; }
+
+    [Required]
+    public string Title { get; set; }
+
+    [Required]
```

Comment:
Wonder if we should add some regex here to match the DFE shiny apps domain just to be extra safe

Reply:
I've added regex checks to `EmbedBlockCreateRequest` and `EmbedBlockUpdateRequest`. Apparently, adding such a check to `EmbedBlock` itself wouldn't get checked.